753 inconsistet fonts and spacing on loop

vitorgomateus opened the pull request:
Fixes #753.
Fixes #754.

**Changes in this request**
- create consistency in the content block width by changing the limit from the `p` rule to the general `.block-paragraph`.
- add `z-index` to the left sidebar so that its content doesn't get visually mixed with the footer.
- new logo's maroon has low contrast with loop's green so it was changed to the all-white logo.

bbusenius reviewed (changes requested):
The logo looks great and I like your use of the consolidated web-resources folder. I'm still seeing font problems on loop though. The lists still aren't constrained and the font size is smaller:

![image](https://github.com/uchicago-library/library_website/assets/7826429/f877d7cc-adb0-450e-8bf1-545c62bc96b7)

I emptied the cache and hard-refreshed so I don't think I'm seeing a caching issue.

vitorgomateus commented:
I didn't notice that the Loop SCSS needs to be compressed separately. I wasn't able to run Gulp, so I made 'compressed' changes manually. I also fixed the difference in font-size and line-height, and added a `overflow-wrap: break-word;` because I saw a long link in this particular example page.

bbusenius commented:
This is looking a lot better. I see the text is constrained now and the line height looks better. I still see a couple of issues though.

1. The list font is still smaller than the paragraph text. I think this is because the paragraph has `font-size: 1.1em` but the list is just inheriting `font-size: 14px` from the body.
2. I think we need to get rid of this `gulp` dependency and move this SASS into the compress framework.

Maybe if we address the first issue in this PR, we can open a new PR for the second. Otherwise, we could address them both here. I don't have a strong preference.

vitorgomateus commented:
It seems these new rules do not apply on the news page type. I'll make new rules to include those as well.
I will create a new issue to get rid of the gulp dependency, or at least create consistency between lib and loop because gulp would be helpful for dev, but it has to work with compress. So I wanna take a little extra time on it.

vitorgomateus commented:
I've tested these pages and it seems the new styles will be consistent:
https://looplet.lib.uchicago.edu/groups/technical-services-policy-group/technical-services-policies/policy-e-holdings-and-urls/
https://looplet.lib.uchicago.edu/departments/administration-and-operations/assessment/
https://looplet.lib.uchicago.edu/documentation/website-authoring/sidebar-promote/library-news/

bbusenius commented:
Nice, this looks good to go!
